opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -229,115 +296,119 @@ impl<R: RuntimeChannel> LogProcessor for BatchLogProcessor<R> {
     }
 }
 
-impl<R: RuntimeChannel> BatchLogProcessor<R> {
-    pub(crate) fn new(mut exporter: Box<dyn LogExporter>, config: BatchConfig, runtime: R) -> Self {
+impl BatchLogProcessor {
+    pub(crate) fn new(mut exporter: Box<dyn LogExporter>, config: BatchConfig) -> Self {
```

Comment:
nit - this will have some conflict with #2374 depending on which is merged first as `dyn exporter` is not possible anymore with native async support. Just to mention, nothing to be done as of now in this PR.